content/en/docs/components/training/legacy-v1/getting-started.md

```diff
@@ -0,0 +1,158 @@
++++
+title = "Getting Started"
+description = "Get started with the Training Operator"
+weight = 30
++++
+
+This guide describes how to get started with the Training Operator and run a few simple examples.
+
+## Prerequisites
+
+You need to install the following components to run examples:
+
+- The Training Operator control plane [installed](/docs/components/training/legacy-v1/installation/#installing-the-control-plane).
```

Comment:
I would move the link here to be around the name of the components and nix "installed" - it doesn't match the previous sentence that leads into it. "You need to install" ... "installed"

Reply:
Actually, we don't want to modify the Training Operator V1 doc as part of this PR, since it focuses on Kubeflow Training V2 doc. Does it look good @vsoch ?